pkg/kudoctl/cmd/test.go

```diff
@@ -165,7 +165,7 @@ For more detailed documentation, visit: https://kudo.dev/docs/testing`,
 
 	testCmd.Flags().StringVar(&configPath, "config", "", "Path to file to load test settings from (must not be set with any other arguments).")
 	testCmd.Flags().StringVar(&crdDir, "crd-dir", "", "Directory to load CustomResourceDefinitions from prior to running the tests.")
-	testCmd.Flags().StringVar(&manifestsDir, "manifests-dir", "", "A directory containing manifests to apply before running the tests.")
+	testCmd.Flags().StringSliceVar(&manifestsDir, "manifests-dir", []string{}, "A directory containing manifests to apply before running the tests.")
```

Comment:
the flag and the help text probably needs to change... `manifests-dirs`?  and "a directory" isn't correct

Reply:
So, I think this should still be singular, since the usage is:

```
kubectl kudo test --manifests-dir ./manifests1/ --manifests-dir ./manifests2/
```

But the setting itself in the configuration should be plural (`ManifestsDirs`) since you do provide it as a list in the YAML:

```
manifestsDirs:
- ./manifests1/
- ./manifests2/
```

Does that seem right? Also, opinion on "manifest" versus "manifests"? `--manifest-dir` or `--manifests-dir` and `ManifestsDirs` or `ManifestDirs`?

Reply:
makes sense on the flag name... `--manifests-dir`  but the underlying is the plural with the option of have 0...1...*

Reply:
I like the cli use of `--manifests-dir`.   I also like the yaml use of `manifestsDirs`.  👍 

Regarding plural on manifest..  I don't have any strong opinion... It is an easy claim that there are usually more than 1 manifest in a dir.. making manifests make sense.   The singular seems easy to communicate and understand that it 1 or more.

Reply:
Cool, thanks! I went with `--manifest-dir` and `ManifestDirs`. I think that's the least confusing.